agents/labjack/labjack_agent.py

```diff
@@ -185,10 +280,14 @@ def make_parser(parser=None):
     pgroup = parser.add_argument_group('Agent Options')
 
     pgroup.add_argument('--ip-address')
-    pgroup.add_argument('--num-channels', default='13')
+    pgroup.add_argument('--active-channels',
+                        default=[str(i+1) for i in np.arange(13)])
```

Comment:
Are there always 13 channels on LabJacks? If we want this to be generic for any LabJack is this a good default?

Reply:
Good point, turns out there are 14 analog inputs on the T7, 12 on the T4. Not sure why 13 was originally hard-coded in there. The LJm module, which we will convert to next PR, only supports the T7 and T4, so that's all we have to care about. Ideally I would be able to automatically detect how many analog channels there are, which is maybe a feature I can do with the LJM module. For now, I just hardcoded two options 'T7-all' and 'T4-all' and set 'T7-all' as the default.

Reply:
Great, the active channels list looks good to me.